### PR TITLE
first draft routes and controllers for users and dogs

### PR DESCRIPTION
**Pull request by lucagi89**
created users and dogs controller with relative relationship in the model

### CODE REVIEW

File: .gitignore
Comment (vincent-chpd) [inline .gitignore:42]: There was already a .env* in line 37, is there a reason why you have added it again?

File: app/views/dogs/create.html.erb
Comment (vincent-chpd) [inline app/views/dogs/create.html.erb:2]: Shouldn't it be new for the view, rather than create? I  thought new was the form and create is only for the POST action.

File: .github/CODEOWNERS
Comment (vincent-chpd) [inline .github/CODEOWNERS:2]: Why did you add your username? lol

**Review by vincent-chpd — COMMENTED** [review]
I have left some of comments before approval

File: .gitignore
Comment (lucagi89) [reply .gitignore:42]: I didn't have it, I had to put my cloudinary key otherwise I couldn't load the app

File: app/views/dogs/create.html.erb
Comment (lucagi89) [reply app/views/dogs/create.html.erb:2]: I imagined that on the user show page we would have a button to add a dog and when you press this button a form pops up, still in the show, where you can create the dog profile without having to load a "new" page

File: .github/CODEOWNERS
Comment (lucagi89) [reply .github/CODEOWNERS:2]: Cause I saw this file and I tought is cool if we add everyone's username

File: app/views/dogs/create.html.erb
Comment (vincent-chpd) [reply app/views/dogs/create.html.erb:2]: okay i see !

File: .github/CODEOWNERS
Comment (vincent-chpd) [reply .github/CODEOWNERS]: No that's the point ! lol I should be the only code owner. Please remove your name and push your branch again 😄

**Review by vincent-chpd — APPROVED** [review]

**Pull request merged**
